piwebapi_samples/Data_Analysis/UploadUtility/Program.cs

```diff
@@ -0,0 +1,293 @@
+﻿using System;
+using System.Xml;
+using System.IO;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace UploadUtility
+{
+    class Program
+    {
+        static readonly string defaultConfigFile = @"..\..\..\..\test_config.json";
```

Comment:
Remind me, why are these files expected to be so much higher in the folder hierarchy?

Reply:
the jupyter notebook also uses this test_config file which is at the root.

Reply:
fixed